tests/e2e/buildah_suite_test.go

```diff
@@ -0,0 +1,337 @@
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"encoding/json"
+	"github.com/containers/image/copy"
+	"github.com/containers/image/signature"
+	"github.com/containers/image/storage"
+	"github.com/containers/image/transports/alltransports"
+	"github.com/containers/image/types"
+	sstorage "github.com/containers/storage"
+	"github.com/containers/storage/pkg/reexec"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+)
+
+var (
+	PODMAN_BINARY      string
```

Comment:
I'm not sure we'll need this.  Do you foresee a use for it?

Reply:
Nope, removed